docs/reference/aggregations/metrics/boxplot-aggregation.asciidoc

```diff
@@ -76,59 +75,56 @@ query for them directly.
 
 ==== Script
 
-The boxplot metric supports scripting. For example, if our load times
-are in milliseconds but we want values calculated in seconds, we could use
-a script to convert them on-the-fly:
+If you need to create a boxplot for values that aren't indexed exactly you
+should create a <<runtime,runtime field>> and get the boxplot of that. For
+example, if our load times are in milliseconds but we want values calculated
+in seconds, we could use a runtime field to convert them on-the-fly:
```

Comment:
```suggestion
If you need to create a boxplot for values that differ from the indexed values,
create a <<runtime,runtime field>> and then compute the boxplot. For
example, if your load times are in milliseconds but you want values calculated
in seconds, use a runtime field to convert values within a query:
```

Reply:
I know that "on-the-fly" was in these docs before you made changes, but we try to avoid idioms like these because they don't translate well beyond English.

Reply:
:+1: